vars/folioexecuteDataMigrationUtils.groovy

```diff
@@ -197,7 +210,7 @@ void sendSlackNotification(String slackChannel, Integer totalTimeInMs = null, Li
     }
 
     try {
-        slackSend(color: karateTestUtils.getSlackColor(buildStatus), message: message, channel: slackChannel)
```

Comment:
are you sure about this change?

Reply:
@eldiiar-duishenaliev it was tested and it works, what is your concern?